src/main/java/org/jabref/logic/formatter/bibtexfields/JournalMedlineAbbreviator.java

```diff
@@ -0,0 +1,52 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import java.util.Objects;
+
+import org.jabref.logic.journals.JournalAbbreviationLoader;
+import org.jabref.logic.journals.JournalAbbreviationPreferences;
+import org.jabref.model.cleanup.Formatter;
+
+public class JournalMedlineAbbreviator implements Formatter {
+
+    private final JournalAbbreviationLoader repostioryLoader;
+    private final JournalAbbreviationPreferences journalAbbreviationPreferences;
+
+    //TODO: How do I pass the prefs at best?
+    public JournalMedlineAbbreviator(JournalAbbreviationLoader repostioryLoader,
```

Comment:
Its probably simpler to only pass the repository as the dependency.

Reply:
Still the problem is that nearly the whole codebase of JabRef is affected as I need to pass the arg from somewhere to the formatter interface